frontend/src/forms/manage-restaurant-form/ManageRestaurantForm.tsx

```diff
@@ -65,6 +68,23 @@ const ManageRestaurantForm = ({ onSave, isLoading }: Props) => {
     },
   });
 
+  useEffect(() => {
+    if (!restaurant) {
+      return;
+    }
+
+    form.reset({
+      restaurantName: restaurant.restaurantName,
+      city: restaurant.city,
+      country: restaurant.country,
+      deliveryPrice: restaurant.deliveryPrice,
+      estimatedDeliveryTime: restaurant.estimatedDeliveryTime,
+      cuisines: restaurant.cuisines,
+      menuItems: restaurant.menuItems,
+      imageUrl: restaurant.imageFile,
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/33#issuecomment-2394670085>#d60155</a><span> - 10/04/2024, 09:21 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'restaurant' prop is correctly added to the component, but there's a potential issue with how it's being used in the useEffect hook.
</div>
<div id="fix">
<b>Fix:</b> Update the useEffect hook to handle all properties of the restaurant object, including the image. Also, ensure that the imageFile field is handled correctly, as it's not a direct property of the restaurant object.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -71,15 +71,16 @@
    useEffect(() => {
      if (!restaurant) {
        return;
      }
 
      form.reset({
        restaurantName: restaurant.restaurantName,
        city: restaurant.city,
        country: restaurant.country,
        deliveryPrice: restaurant.deliveryPrice,
        estimatedDeliveryTime: restaurant.estimatedDeliveryTime,
        cuisines: restaurant.cuisines,
        menuItems: restaurant.menuItems,
 -      imageUrl: restaurant.imageFile,
 +      imageUrl: restaurant.imageUrl,
 +      // Note: imageFile is not set here as it's not part of the existing restaurant data
      });
    }, [restaurant, form]);
```
</div>
</details>
</div>
<br>


<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The imageUrl field is being populated with restaurant.imageFile, which may not be the correct property.
</div>
<div id="fix">
<b>Fix:</b> Change 'imageUrl: restaurant.imageFile,' to 'imageUrl: restaurant.imageUrl,'
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -84,1 +84,1 @@
 -      imageUrl: restaurant.imageFile,
 +      imageUrl: restaurant.imageUrl,
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged